src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1287,6 +1287,15 @@
   <data name="PreferStreamAsyncMemoryOverloadsMessage" xml:space="preserve">
     <value>Change the '{0}' method call to the overload that receives a '{1}'.</value>
   </data>
+  <data name="ForwardCancellationTokenToAsyncMethodsDescription" xml:space="preserve">
+    <value>Consider forwarding the 'CancellationToken' parameter to methods that take one to ensure the operation cancellation notifications gets properly propagated.</value>
+  </data>
+  <data name="ForwardCancellationTokenToAsyncMethodsMessage" xml:space="preserve">
+    <value>Consider forwarding the '{0}' parameter to the '{1}' method.</value>
```

Comment:
Should we add " or pass in CancellationToken.None explicitly to indicate intentionally not propagating the token." to help folks realize how to suppress the diagnostic?

Reply:
I think that's a good idea.

Reply:
Agreed. We should also make the message assertive `Forward the '{0}' parameter ...`

Reply:
I included the suggested changes to the strings. Let me know if there's anything else we should mention or remove from them.